instrumentation/github.com/gofiber/fiber/README.md

```diff
@@ -0,0 +1,41 @@
+### OpenTelemetry for [Fiber](https://gofiber.io/)
```

Comment:
This markdown is not properly formatted. E.g. this line should have `#` instead of `###`

I would also add a hyperlink to the `example` directory instead of putting the code here as it may easily get out-of-sync.

I think it is very good that you want to provide `README.md` 👍 

However, because no other instrumentation has a `README.md` and also I want to propose some documentation guidelines as part of https://github.com/open-telemetry/opentelemetry-go/issues/2046, I suggest to simply remove this file 😉